src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
+    static final int COUNT_OF_FIGURES = 5;
```

Comment:
According to the checklist, static methods and fields are generally considered bad practice in OOP design. Consider refactoring the code to avoid using static fields unless they are constants, and this field should be a constant with the appropriate naming convention.